azure-pipelines/e2e_projects/applocal/basic/installed/bin/mylib.cpp

```diff
@@ -1,7 +1,8 @@
 #define MYLIB_EXPORTS 1

```

Comment:
I'm a bit confused about why these are in an `installed/bin` folder. Is it to reflect that they will be transformed into some `mylib.lib` in this subpath under the testing root?

Reply:
Because when you run cl, it puts the DLL in the same location, and when installed these go into `${CURRENT_INSTALLED_DIR}/bin`